src/test/regress/expected/multi_join_pruning.out

```diff
@@ -75,6 +75,22 @@ DEBUG:  join prunable for intervals [13473,14947] and [1,5986]
      |    
 (1 row)
 
+-- Make sure that we can handle filters without a column
```

Comment:
The output should be the same except for the count distinct one, which should turn back to normal.